android/app/src/main/java/app/organicmaps/bookmarks/Holders.java

```diff
@@ -406,11 +406,10 @@ void bind(@NonNull SectionPosition position,
       final long trackId = sectionsDataSource.getTrackId(position);
       Track track = BookmarkManager.INSTANCE.getTrack(trackId);
       mName.setText(track.getName());
-      mDistance.setText(new StringBuilder().append(mDistance.getContext()
```

Comment:
1. Was it done deliberately to speed up the code? Is there any comment in git blame?
2. Does it make sense to cache .getString(R.string.length) result here and maybe in some other places, where it is called repeatedly for many objects?

Reply:
Nothing in git blame
It's the only place using `.getString(R.string.length)`